components/GptSection.jsx

```diff
@@ -0,0 +1,42 @@
+function GptSection() {
+  return (
+    <>
+    <div className="m-10  inline-block  bg-gradient-to-br from-color-blog to-blue-400">
+
+    <span className="flex justify-between">
+    <div className="pl-8 pt-8 text-3xl font-bold text-white w-1/2">What is GPT-3</div>
+    
+    <div className="pl-3 pt-9 pr-7 text-xs w-1/2 text-color-text">We so opinion friends me message as delight. 
+    Whole front do of plate heard oh ought. His defective nor convinced residence own. 
+    Connection has put impossible own apartments boisterous.
+     At jointure ladyship an insisted so humanity he. 
+     Friendly bachelor entrance to on by.</div>
+    </span>
+    <br/>
+    <br/>
+    <br/>
+    <br/>
+    <span className="p-8 flex justify-between">
+    <h3 className=" text-3xl bg-gradient-to-r from-purple-500 to-orange-400 bg-clip-text text-transparent w-1/3">The possibilities are beyond your imagination</h3>
+    <h6 className=" text-orange-400 text-sm ">Explore The Library</h6>
+    </span>
+    
+    <br/>
+    <br/>
+    <div className="p-8 flex justify-between font-bold text-white">
+    <h4 className="pr-25">Chatbots</h4>
```

Comment:
try to make a div for every h1 and the message below change it to p inside the same div